source/features/github-bugs.css

```diff
@@ -1,22 +1,3 @@
-/* Place Releases/Tags selector on the same line as `Draft new release` on releases page #2325 */
-.subnav .d-flex.flex-md-row.flex-justify-between.flex-md-items-center {
-	float: left;
-}
-
-/* Align fork popup's header #3352 */
-[aria-label^='Fork'] .Box-title {
-	position: static; /* The title wrongly overlaps the Close button, making it impossible to click */
-	min-height: 0;
-	margin: -5px 0;
-}
```

Comment:
Test URL: the "Fork to" popup

Fixed by GitHub:

![image](https://user-images.githubusercontent.com/46634000/153644990-62b1640a-c019-401c-985c-3ff42cc929a1.png)